packages/recipe/src/components/EzLineChart/__tests__/EzLineChart.test.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {render} from '@testing-library/react';
+import {axe} from '../../../../test-utils';
```

Comment:
In order for the tests to use themes correctly, we need to import `render` from our `test-utils` file. Probably not an issue here, but if we add any theme-related tests in the future, it will be.

```suggestion
import {axe, render} from '../../../../test-utils';
```